sections/02_intro.md

```diff
@@ -1,97 +1,155 @@
 ## Introduction
 
-### Potential writing prompt
```

Comment:
One suggestion: once we decide on other main subsections, maybe we can come up with one central guiding story here to motivate/orient the whole work, and to integrate different subsections that seem isolated. i.e., what is the ultimate vision? Maybe a patient comes into hospital, gets his molecules measured, grants access to his EHR. Then our deep learning based engine/knowledge-base gives diagnosis/categorization/medication advice. Meanwhile, his data will be "shared" and recorded into our system, allowing refinement of our deep learning system.